vmss/vmss-dump-using-module.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
the naming is confusing. it's hard to understand what's different between this one and vmss_dump.yml. I suggest you to add a readme file to describe what  you want to say by adding those yaml files.

Reply:
maybe some playbooks are not necessary, that's another issue